code/src/java/plugin/lsttokens/ModifyLst.java

```diff
@@ -58,40 +56,71 @@ public String getTokenName()
 	@Override
 	public ParseResult parseNonEmptyToken(LoadContext context, VarHolder obj, String value)
 	{
-		//TODO These instanceof checks will fail - the VarHolder is a proxy :(
-		if (obj instanceof Ungranted)
+		/*
+		 * TODO Need to check the object type of the VarHolder to make sure it is legal.
```

Comment:
Not a big fan of TODO in source alone. This should be recorded in a JIRA issue so it doesn't fall thru the cracks and referenced here.